indico/modules/events/abstracts/views.py

```diff
@@ -45,7 +45,8 @@ def getCSSFiles(self):
         return (WPConferenceModifBase.getCSSFiles(self) +
                 self._asset_env['markdown_sass'].urls() +
                 self._asset_env['selectize_css'].urls() +
-                self._asset_env['abstracts_sass'].urls())
+                self._asset_env['abstracts_sass'].urls() +
+                self._asset_env['contributions_sass'].urls())
```

Comment:
It's for the "Contribution types" dialog. You really think I should move the code to `eventmanagement.scss`? We also import the file in `WPDisplayAbstractsBase`, not just `WPManageAbstracts`...